third_party/txt/src/skia/paragraph_skia.cc

```diff
@@ -180,6 +202,14 @@ class DisplayListParagraphPainter : public skt::ParagraphPainter {
     return path;
   }
 
+  bool ShouldRenderAsPath(const DlPaint& paint) const {
+    // Text with non-trivial color sources or stroke paint mode should be
```

Comment:
Thanks for adding context.

Is this for performance or correctness or both? Could you note that?

Reply:
Added correctness information.